copier/main.py

```diff
@@ -362,6 +363,17 @@ def answers_relpath(self) -> Path:
         """
         return self.answers_file or self.template.answers_relpath
 
+    @cached_property
+    def config_file(self) -> Path:
+        """Obtain the proper path for the config file.
+
+        It comes from:
+
+        1. User choice.
+        2. Copier default.
+        """
+        return self.config_file
+
```

Comment:
This code is buggy because the computed property and the field are named the same. Just remove it.

Reply:
Removed